src/W/Message.elm

```diff
@@ -4,6 +4,7 @@ module W.Message exposing
     , primary, secondary, success, warning, danger, color
     , href, onClick
     , htmlAttrs, noAttr, Attribute
+    , borderWidth, noBorder
```

Comment:
We need to add the new public functions to our module docs like this:

```elm
@docs borderWidth, noBorder
```

(or add it as part of another list of docs if it makes sense!)